src/card.tsx

```diff
@@ -44,34 +58,39 @@ const Card = ({ firstName, lastName, id, status, joinDate, ot }: User): JSX.Elem
             <img src={logo} className="App-logo" alt="logo" />
           </div>
         </div>
+        <div>
+          <DigitalClock />
+        </div>
+        <div className="px-8 pt-2 pb-4">
+          <img src={`data:image/png;base64,${qrCode}`} />
+        </div>
         <div className="border-t border-gray-200">
           <dl>
             <CardLine
-              label="Imię i nazwisko" 
-              value={<span className="font-bold">{fullName}</span>} 
-              isEven 
-            />
-            <CardLine 
-              label="Numer" 
-              value={id} 
-            />
-            <CardLine 
-              label="Składka opłacona" 
-              value={<span className="text-green-600 font-bold">TAK</span>} 
-              isEven 
-            />
-            <CardLine 
-              label="Data przystąpienia" 
-              value={new Intl.DateTimeFormat(LOCALE).format(Number(joinDate) * 1000)} 
+              label="Imię i nazwisko"
+              value={<span className="font-bold">{fullName}</span>}
+              isEven
             />
+            <CardLine label="Numer" value={pspdId} />
             <CardLine
-              label="Oddział"
-              value={ot}
+              label="Składka opłacona"
+              value={<span className="text-green-600 font-bold">TAK</span>}
               isEven
             />
+            <CardLine
+              label="Data przystąpienia"
+              value={new Intl.DateTimeFormat(LOCALE).format(
```

Comment:
Dobrze, by było spójnie z innymi datami.

```suggestion
              value={new Intl.DateTimeFormat(LOCALE, options).format(
```